src/components/edit-movie/edit-movie.tsx

```diff
@@ -50,16 +51,40 @@ export const EditMovie = ({ title, movie, onClose, action }: Props) => {
   const handleReset = () => {
     onClose();
   };
-  const required = (value: any) => (value ? undefined : 'Required');
+  const getValidator = (value: string | number | string[] | undefined) =>
+    value ? undefined : 'Required';
 
-  const initialValues = {
-    title: movie?.title,
-    release_date: movie?.release_date,
-    poster_path: movie?.poster_path,
-    vote_average: movie?.vote_average,
-    genres: movie?.genres,
-    runtime: movie?.runtime,
-    overview: movie?.overview,
+  const initialValues = useMemo(
+    () => ({
+      title: movie?.title,
+      release_date: movie?.release_date,
+      poster_path: movie?.poster_path,
+      vote_average: movie?.vote_average,
+      genres: movie?.genres,
+      runtime: movie?.runtime,
+      overview: movie?.overview,
+    }),
+    [movie],
+  );
+
+  const TextFieldAdapter = ({
+    input,
+    meta,
+    placeholder,
+    className,
+    type = 'text',
+  }: FieldRenderProps<string | number>) => {
+    return (
+      <>
+        <input
+          {...input}
+          className={classNames(className, styles.input)}
+          type={type}
+          placeholder={placeholder}
+        />
+        {meta.error && meta.touched && <span>{meta.error}</span>}
+      </>
+    );
```

Comment:
Well, I would prefer to move it to separate component to make more reusable across app. Take a look at my example [here](https://github.com/dotnet-fizzyy/Diploma/tree/V2/web/src/components/common): `common` folder is a place to store all reusable component across app, form component are not exception :)